mta-both.yaml

```diff
@@ -0,0 +1,252 @@
+_schema-version: "3.2"
+ID: capire.sflightNJ
+version: 1.0.0
+description: "CAP sflight demo (Node&Java)"
+
+parameters:
+  enable-parallel-deployments: true
+
+build-parameters:
+  before-all:
+    - builder: custom
+      commands:
+        # hana is part of node build (cannot be separated as each of them first deletes gen folder)
+        - npx -p @sap/cds-dk cds build --profile production,node
+        - npx -p @sap/cds-dk cds build --profile production,java
+
+
+modules:
+  # --------------------- SERVER MODULES -----------------------
+  - name: sflightN-srv
+  # ------------------------------------------------------------
+    type: nodejs
+    path: gen/srv
+    requires:
+      - name: sflightN-db
+      - name: sflight-uaa
+    provides:
+      - name: srvN-api # required by consumers of CAP services (e.g. approuter, destinations)
+        properties:
+          srv-url: ${default-url}
+    build-parameters:
+      builder: npm-ci
+      ignore: ["node_modules/"]
+
+  # ------------------------------------------------------------
+  - name: sflightJ-srv
+  # ------------------------------------------------------------
+    type: java
+    path: srv
+    properties:
+      SPRING_PROFILES_ACTIVE: cloud
+      JBP_CONFIG_COMPONENTS: "jres: ['com.sap.xs.java.buildpack.jdk.SAPMachineJDK']"
+      JBP_CONFIG_SAP_MACHINE_JRE: '{ jre: { version: 11.+ } }'
+    requires:
+      - name: sflightJ-db
+      - name: sflight-uaa
+    provides:
+      - name: srvJ-api # required by consumers of CAP services (e.g. approuter, destinations)
+        properties:
+          srv-url: ${default-url}
+    build-parameters:
+      builder: custom
+      commands:
+        - mvn clean package -DskipTests=true
+      build-result: target/*-exec.jar
+      ignore: ["node_modules/"]
+
+
+  # -------------------- SIDECAR MODULES HDI DEPLOY ------------
+  - name: sflightN-db-deployer
+  # ------------------------------------------------------------
+    type: hdb
+    path: gen/db
+    parameters:
+      buildpack: nodejs_buildpack
+    requires:
+      - name: sflightN-db
+      - name: sflight-uaa
+    build-parameters:
+      ignore: ["node_modules/"]
+
+  # ------------------------------------------------------------
+  - name: sflightJ-db-deployer
+  # ------------------------------------------------------------
+  # can we avoid the 2nd (unnecessary) npm install ??? it doesn't do much, but anyway ...
```

Comment:
You could have `sflightJ-db-deployer` depend on `sflightN-db-deployer` and disable the build command
```yml
build-parameters:
  requires: sflightN-db-deployer
  builder: custom
    commands: []
```